test_shim_post fails 1 case on my laptop (ubuntu 23.04)

```

blacklab% make test_shim_post
Makefile:50: warning: ignoring prerequisites on suffix rule definition
./shim_post.sh >shim_post.log 2>&1
#python3 ./check_shim_post.py shim_post.log exit_on_bad
python3 ./check_shim_test.py shim_post.log
+.
 .
 .
2023-05-09 16:47:42,070 [INFO] published: { "pubTime":"20230509204742.0688027", "baseUrl":"file:/home/peter", "relPath":"test/hoho_my_darling.txt", "topic":"v03.post.test", "integrity":{  "method" : "sha512", "value" : "vuIxy9N84gi1FNtJ++ahgMQN+gAoyf8evK6GIAAgsva4IFlqL+37NVNrjImaMa/jtrKN6Qi+W/Ahcs\nqKMRvz0w=="  } , "mtime":"20230509204742.0619578", "atime":"20230509204742.0619578", "mode":"0664", "size":"40238", "toto":"pig"}

RESULT: BAD! binary crashed. 
RESULT: Good! 1 sha512 posts from 070 cp command. as expected
+ touch hihi
 .
 .
 .
RESULT: Good! 2 remove posts from 210 removing two files. as expected
RESULT: comment 220 shim test posting end
posts: {'sha512': 7, 'directory': 3, 'rename': 5, 'rmdir': 1, 'link': 1, 'remove': 6}
RESULT: summary: good: 21, bad 1, total 21
make: *** [Makefile:127: test_shim_post] Error 1
blacklab% 
```

so one error in the tests, with a binary crashing.   Don't know if it is the laptop, or the OS, or something in the environment...   did a git bisection, and the issues is not new, it crashed in all versions going back three months.

looking at the log of the script itself:
```
  .
  .
  .
023-05-09 16:47:42,071 [DEBUG] realpath_adjust -2, /home/peter/Sarracenia -> /home/peter/Sarracenia
2023-05-09 16:47:42,071 [DEBUG] sr_isMatchingPattern, testing string: /home/peter/Sarracenia/metpx-sr3c/shim_post.log vs mask: accept /home/peter/Sarracenia/metpx-sr3c/.* next=0x562c7a0cbba0
2023-05-09 16:47:42,071 [DEBUG] sr_isMatchingPattern: /home/peter/Sarracenia/metpx-sr3c/shim_post.log matched mask: accept /home/peter/Sarracenia/metpx-sr3c/.*
SR_SHIMDEBUG 1 1142830 0.0713236 srshim_realpost accepted... /home/peter/Sarracenia/metpx-sr3c/shim_post.log now
SR_SHIMDEBUG 1 1142830 0.0713277 srshim_realpost rejecting should_not_post... /home/peter/Sarracenia/metpx-sr3c/shim_post.log
SR_SHIMDEBUG 3 1142830 0.071331 shim re-enabled after post of /home/peter/Sarracenia/metpx-sr3c/shim_post.log
SR_SHIMDEBUG 5 1142830 0.0713353  fclose 0x7f8e68df76a0 fd=2 fdstat=1758506608, starting
Segmentation fault (core dumped)
+ echo #test 1 sha512 080 touch command
#test 1 sha512 080 touch command
+ touch hihi
 startup SR_
  .
 .
 .

```

no ideas what is special about this laptop... noting for later investigation.

--- a/libsr3shim.c
+++ b/libsr3shim.c
@@ -1505,45 +1505,6 @@ ssize_t sendfile(int out_fd, int in_fd, off_t * offset, size_t count)
 	return (status);
 }
 
-static int copy_file_range_init_done = 0;
-typedef ssize_t (*copy_file_range_fn) (int, off_t *, int, off_t *, size_t, unsigned int);
-static copy_file_range_fn copy_file_range_fn_ptr = NULL;
-
-ssize_t copy_file_range(int fd_in, loff_t * off_in, int fd_out,
-			loff_t * off_out, size_t len, unsigned int flags)
-{
-	ssize_t status;
-	char fdpath[32];
-	char real_path[PATH_MAX + 1];
-	char *real_return;
-
-	if (!copy_file_range_init_done) {
-		setup_exit();
-		copy_file_range_fn_ptr = (copy_file_range_fn) dlsym(RTLD_NEXT, "copy_file_range");
-		copy_file_range_init_done = 1;
-	}
-	status = copy_file_range_fn_ptr(fd_in, off_in, fd_out, off_out, len, flags);
-	if (shim_disabled)
-		return (status);
-
-	snprintf(fdpath, 32, "/proc/self/fd/%d", fd_out);
-	real_return = realpath(fdpath, real_path);
-
-	sr_shimdebug_msg(1, "copy_file_range to %s\n", real_path);
-
-	if (!real_return)
-		return (status);
-	if (!strncmp(real_path, "/dev/", 5))
-		return (status);
-	if (!strncmp(real_path, "/proc/", 6))
-		return (status);
-
-	shimpost(real_path, 0);
-
-	clerror(status);
-	return (status);
-}
-
 int close(int fd)
 {
 

